docs/build-for-developers/cli-usage.md

```diff
@@ -48,16 +49,52 @@ openfn path/to/job.js -a http@2.0.0
 openfn path/to/job.js -a http=/repo/openfn/adaptors/my-http-build
 ```
 
-**Use the build in the adaptors monoreo:**
+**Use the local adaptors monorepo build:**
 
 ```bash
 openfn path/to/job.js -ma http
 ```
 
-Set a path to the monorepo with the env var OPENFN_REPO_DIR (eg, `OPENFN_REPO_DIR=~/openfn/repo openfn job.js -ma http`).
+You must set the path to the monorepo in the env var OPENFN_ADAPTORS_REPO. For
+example:
+
+```bash
+OPENFN_ADAPTORS_REPO=~/openfn/adaptors openfn job.js -ma http
```

Comment:
@josephjclark  this`OPENFN_ADAPTORS_REPO=~/openfn/adaptors openfn job.js -ma http` should be `OPENFN_ADAPTORS_REPO=~/openfn/adaptors`

Reply:
No, it's correct as it is.

This runs a job with the monorepo path set in a var. It's an actually useful command.

If we just did `OPENFN_ADAPTORS_REPO=~/openfn/adaptors` then nothing would happen.

We could do `export OPENFN_ADAPTORS_REPO=~/openfn/adaptors` but that I think will only work within that session? So it's not terribly helpful.

This is why I added the line about adding to your `.profile`

Reply:
Ooh okay, got you